.github/workflows/default.yaml

```diff
@@ -44,9 +44,8 @@ jobs:
 
       - run: ./bootstrap.sh
       - run: ./configure --with-openssl
-      - run: make -j2
       - run: |
-          sudo make install
+          sudo make -j`nproc` install
```

Comment:
This change request assumes that old "make [all]" command was restored and this line is specific to Squid installation.

I am not violently against adding `-jn` to "make install", but it feels wrong (unless it buys us significantly faster builds) because most folks are not going to use this kind of installation command and because errors from parallel installs are more difficult to interpret. In other words, I think this optimization needs to be justified.